code/__HELPERS/colors.dm

```diff
@@ -17,6 +17,18 @@
 
 	return final_color
 
+/// Given a color in the format of "#RRGGBB" or "#RRGGBBAA", gives back a 4 entry list with the number values of each
+/proc/split_color(color)
```

Comment:
why was this added if it's unused? helper?

Reply:
Oh, yeah I ended up not needing it but it's still generic and potentially useful so I left it